src/framing/mod.rs

```diff
@@ -63,7 +63,7 @@ impl MLSPlaintext {
         mls_plaintext
     }
     // XXX: Only used in tests right now.
-    pub(crate) fn from_bytes(bytes: &[u8]) -> Result<Self, CodecError> {
```

Comment:
If it's really used in tests, add a `#[cfg(tests)]` to make the warning go away.